.github/workflows/test.yml

```diff
@@ -33,10 +33,4 @@ jobs:
         run: npm run build
 
       - name: Run unit tests
-        run: npm test
-        env:
-          CI: true
-          NODE_ENV: 'test'
-
-      - name: Report test coverage
```

Comment:
I noticed you removed `npm test`, and just replaced it with the reporting stage (`ArtiomTr/jest-coverage-report-action@v2.1.1`). Does that automatically run the tests?

Reply:
It does, twice. Once to get one batch of coverage, and another time on the target branch to get the other batch, to diff the coverage for the chart. If the tests fail, then it blocks the pipeline (as it should).

Reply:
I'd previously assumed that it gets coverage from our own test run, but no, it runs its own.

Reply:
You can see this in [this PR's previous CI run](https://github.com/BYU-CS-Discord/CSBot/actions/runs/4483065715/jobs/7881896098), under the "Run unit tests" action.

Reply:
Actually, we've disabled this call anyway, lol